google/gax/grpc.py

```diff
@@ -108,15 +108,14 @@ def create_stub(generated_create_stub, service_path, port, ssl_creds=None,
     """
     if channel is None:
         if ssl_creds is None:
-            ssl_creds = implementations.ssl_channel_credentials(
-                None, None, None)
+            ssl_creds = grpc.ssl_channel_credentials(None, None, None)
```

Comment:
These are keyword arguments that default to `None`: https://github.com/grpc/grpc/blob/01da196879cb69b3c56b3bdcff6f0194ab5a190d/src/python/grpcio/grpc/__init__.py#L1029

Best to not pass anything at all.